build-tools/packages/build-cli/src/lib/dangerfile.ts

```diff
@@ -24,21 +23,17 @@ const adoConstants = {
 };
 
 const localReportPath = "./artifacts/bundleAnalysis";
-let logger: Logger;
 
 export async function dangerfile() {
```

Comment:
Could we add some documentation to this? I can sort of guess as to what it is for / how it is used, but some explicit docs would be helpful.